samples/esb/esb_ptx/README.rst

```diff
@@ -1,13 +1,34 @@
-.. _esb_prx_ptx:
+.. _esb_ptx:
 
-Enhanced ShockBurst: Transmitter/Receiver
-#########################################
+Enhanced ShockBurst: Transmitter
+################################
 
 .. contents::
    :local:
    :depth: 2
 
-The Enhanced ShockBurst Transmitter/Receiver sample shows the basic steps that are needed to transmit and receive packets using :ref:`ug_esb`.
+This sample shows the Enhanced ShockBurst Transmitter application.
+It includes the basic steps that are needed to transmit and receive packets using :ref:`ug_esb`.
+Follow the instructions and use the files from this page, and the :ref:`esb_prx` sample.
+
+Requirements
+************
+
+.. note::
+   Follow the steps from and include the :ref:`esb_prx` sample with this sample.
```

Comment:
I am not sure this note is needed. The esb_prx sample is needed for testing the esb_ptx sample.